tailwind.config.js

```diff
@@ -3,7 +3,12 @@ module.exports = {
   purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {},
+    extend: {
+      backgroundImage: theme => ({
+        'ell-back': "url('/src/images/menuWall.svg')",
```

Comment:
what is the main reason for this?

Reply:
to stick to tailwind lib , so importing the bg image to this file instead of using inline styling using <style >, with this change all you have to do is to use this "bg-ell-back" inside the <className>